experimental/docker-compose/README.md

```diff
@@ -37,7 +37,45 @@ Under Kitematic, click on the application 'deck' -> settings -> ports. Click on
 
 <img src="https://cloud.githubusercontent.com/assets/74310/11158618/4bba7122-8a0e-11e5-83b6-8ff2297562b2.png"/>
 
-Alternatively, you can just enter ```DOCKER_IP=`docker-machine ip default` && open http://$DOCKER_IP:9000```
+Alternatively, you can just enter ```open http://`docker-machine ip default`:9000```
+
+## Using locally built images
+
+Sometimes it is useful to use the docker compose configuration to test changes that span multiple services locally.
+
+We provide a development configuration that will build a service from source instead of downloading an image from the remote repository. 
+
+The setup is similar to the local configuration outlined at the root of this project.
+
+* Create a directory called build that sits at the same level as the spinnaker directory ( ../../../build/ from the docker-compose directory ).
```

Comment:
Maybe say "is a sibling to the spinnaker directory" instead of "sits at the same level as"? Could probably drop the ../../../build stuff then.